packages/node_modules/ciscospark/src/ciscospark.js

```diff
@@ -51,7 +51,7 @@ const CiscoSpark = SparkCore.extend({
 CiscoSpark.version = PACKAGE_VERSION;
 
 /**
- * Create a new ciscospark instance
+ * Create a new CiscoSpark instance
```

Comment:
This should stay capitalized and actually be `Create a new {@link CiscoSpark} instance`

`CiscoSpark` is the class name `ciscospark` is a potential name of the instance.